content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Programmatically Access Log Data Using the Logs Search API
+kind: guide
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Log Explorer"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Search Syntax"
+- link: "/logs/search_syntax/"
+  tag: "Documentation"
+  text: "Learn more about syntax for Logs Search API"
+---
+
+
+## Overview
+
+Use the [Logs Search API][1] to programmatically access your log data and execute queries.
+
+The following examples are covered in this guide:
+
+* [Basic search](#basic-search)
+* [Sort by facet or timestamp](#sort-by-facet-or-timestamp)
+* [Limit the number of results retrieved](#limit-the-number-of-results-retrieved)
+* [Pagination](#pagination)
+
+## Prerequisites
+
+- An API key and an application key from an admin user is required to use the API. These are available in your [Datadog account API key page][2]. Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key.
+
+- This guide features `curl` examples. Install [curl][3] if you do not have it installed, or reference additional language examples for this API endpoint in the [API documentation][1].
+
```

Comment:
Let's include a note that:
* by Default, all users (Admin, Standard, Read-Only) have access to all log data.
* the search results might be restricted based on the permissions of the user owning the app-key and link to the permission section https://docs.datadoghq.com/account_management/rbac/permissions/?tab=ui#log-data-access